examples/py/bybit-USDC-create-option-order.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from pprint import pprint
+
+root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(root + '/python')
+
+import ccxt  # noqa: E402
+
+
+exchange = ccxt.bybit ({
+    'apiKey': 'YOUR_API_KEY',
+    'secret': 'YOUR_API_SECRET',
+    'defaultType': 'option',
+    # 'verbose': True,  # for debug output
+})
+
+# BTC/USD:USDC-YYMMDD-STRIKE-C
```

Comment:
I'm thinking you should specify what `C` is at the end, and should it maybe be something like `BASE/QUOTE:SETTLE-YYMMDD-STRIKE-C (end with C for call, end with P for put)`?